docs/migration_guide.rst

```diff
@@ -59,28 +64,36 @@ Breaking changes
 4. All error messages have been standardized with capitalization at the beginning and a full stop at the end.
 
 |
+.. raw:: html
+
+   <hr>
 |
 
+**********************
 0.12.0 Migration guide
-======================
+**********************
 
```

Comment:
The docs build failed because of warnings.
Maybe we could do
```
|

----

|
```

I think these lines would be less visible though, so maybe we could also remove "Migration guide" from every section and leave just the version, so it would be more visible.